manifest_staging/charts/secrets-store-csi-driver/templates/role-psp.yaml

```
@@ -0,0 +1,13 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: secrets-store-csi-driver-role-psp
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: [ 'extensions' ]
+    resources: [ 'podsecuritypolicies' ]
+    verbs: [ 'use' ]
+    resourceNames:
+      - secrets-store
```

```suggestion
      - secrets-store-csi-driver-psp
```

This should match the psp name from above